imin.jsonld

```diff
@@ -179,6 +179,15 @@
       "@type": "oa:OrderItemStatus",
       "label": "CustomerCancellationPending",
       "comment": "Customer cancellation pending but not yet complete"
+    },
+    {
+      "@id": "imin:NotYetImplementedError",
+      "@type": "Class",
+      "label": "NotYetImplementedError",
+      "subClassOf": [
+        "oa:OpenBookingError"
+      ],
+      "comment": "imin NotYetImplemetedError subclassing OpenActive's OpenBookingError"
```

Comment:
Comment should explain what the class/property/error is for, rather than describing what the JSON-LD markup is doing
 
```suggestion
      "comment": "Error returned when requested functionality is not yet implemented"
```